examples/widgets/roller/lv_example_roller_3.py

```diff
@@ -29,6 +26,15 @@ def __init__(self):
         #if LV_FONT_MONTSERRAT_22
         #    lv_obj_set_style_text_font(roller1, &lv_font_montserrat_22, LV_PART_SELECTED);
         #endif
+        try:
+            roller1.set_style_text_font(lv.font_montserrat_22)
+        except:
+            fs_drv = lv.fs_drv_t()
+            fs_driver.fs_register(fs_drv, 'S')
+            print("montserrat-22 not enabled in lv_conf.h, dynamically loading the font")
+            font_montserrat_22 = lv.font_load("S:" + "../../assets/font/montserrat-22.fnt")
+            roller1.set_style_text_font(font_montserrat_22,lv.PART.SELECTED)
```

Comment:
Any reason why `lv.PART.SELECTED` is specified on the dynamic codepath but not the static one?

Reply:
The reason is that it's a bug! Corrected!